stl/inc/xtr1common

```diff
@@ -168,7 +168,7 @@ _INLINE_VAR constexpr bool disjunction_v = disjunction<_Traits...>::value;
 
 template <class _Ty, class... _Types>
 _INLINE_VAR constexpr bool _Is_any_of_v = // true if and only if _Ty is in _Types
-    disjunction_v<is_same<_Ty, _Types>...>;
+    (is_same_v<_Ty, _Types> || ...);

```

Comment:
Fold expressions are a C++17 feature, but the STL must work in C++14 mode. 
The function '_is_any_of_v' starts with an underscore and may be used for internal C++14 machinery (although I am not sure if it is actually used).

Reply:
Tests failed.

The machinary is actually used in C++14 mode.

From the CI:

```
D:\build\out\inc\xtr1common(171): error C7518: fold expressions require at least '/std:c++17'
D:\build\out\inc\xlocnum(1216): note: see reference to variable template 'const bool _Is_any_of_v<char,char,wchar_t>' being compiled
D:\build\out\inc\ostream(293): note: see reference to class template instantiation 'std::num_put<_Elem,std::ostreambuf_iterator<_Elem,_Traits>>' being compiled
        with
        [
            _Elem=char,
            _Traits=std::char_traits<char>
        ]
```

Sorry, I think your current PR can't be merged...

If you want to create a PR you could take the issue: https://github.com/microsoft/STL/issues/3929